spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataflowOAuthSecurityConfiguration.java

```diff
@@ -32,9 +36,42 @@
 @Conditional(OnOAuth2SecurityEnabled.class)
 public class DataflowOAuthSecurityConfiguration extends OAuthSecurityConfiguration {
 
+	private static final String SWAGGER_UI_CONTEXT = "/swagger-ui/**";
+
+	@Value("${springdoc.api-docs.path:/v3/api-docs}")
```

Comment:
I think the security concerns are very appropriate about this feature. Because we needed the API to be available public I implemented it that way, but the changes could also be easily removed. I guess the server needs to be requested with a bearer token / with a previous login if security is enabled in this case.

If this would be a better way to provide the feature for the majority of SCDF users I would be fine 👍  

I thinks `springdoc.show-actuator` is only be considered when the actuator endpoints are activated via configuration in the application.yaml

Reply:
> I just checked if it is possible to remove the security configuration. 

Did you move the code to a WebSecurityCustomizer? I did not mean to get rid of it entirely, just move it out of here.

Reply:
> I thinks springdoc.show-actuator is only be considered when the actuator endpoints are activated via configuration in the application.yaml

I too think we are safe here but just wanted to call it out as an area we would need to verify closely.

Reply:
> Because we needed the API to be available public I implemented it that way, 

Yep, got it and I understand why you did it. I would have done the same thing. And this may be the way we end up going. Lets see what the rest of the team chimes in on etc. For now, lets assume we will leave it open and just move this code into a separate customizer. Wdyt?

Reply:
> Did you move the code to a WebSecurityCustomizer? I did not mean to get rid of it entirely, just move it out of here.
I tested it locally so no changes to this branch were made. 😁 

I moved the code to a WebSecurityCustomizer and restored `DataflowOAuthSecurityConfiguration` to the original implementation.